aptos-move/framework/aptos-framework/releases/artifacts/current/build/AptosFramework/sources/Token.move

```diff
@@ -222,18 +227,26 @@ module AptosFramework::Token {
         let gallery = &mut borrow_global_mut<Gallery<TokenType>>(account_addr).gallery;
         if (Table::contains_key(gallery, &token.id)) {
             let current_token = Table::borrow_mut(gallery, &token.id);
-            current_token.balance = current_token.balance + token.balance
+            merge_token(token, current_token);
```

Comment:
no events?

Reply:
I think this is a great time to talk about what are meaningful events. Would we want every action to result in an Event? Should the data changed on change be reflected completely in events? E.g., in the past, folks often shared all data when there was an update, but that made tracing the blockchain complete by following our event stream but I doubt that would be consistently held. It also causes a doubling in storage costs.